src/gui/EntryPreviewWidget.cpp

```diff
@@ -72,8 +72,17 @@ EntryPreviewWidget::EntryPreviewWidget(QWidget* parent)
     connect(m_ui->toggleEntryNotesButton, SIGNAL(clicked(bool)), SLOT(setEntryNotesVisible(bool)));
     connect(m_ui->toggleGroupNotesButton, SIGNAL(clicked(bool)), SLOT(setGroupNotesVisible(bool)));
     connect(m_ui->entryTabWidget, SIGNAL(tabBarClicked(int)), SLOT(updateTabIndexes()), Qt::QueuedConnection);
+    // Prevent the url from being focused after clicked to allow the Copy Password button to work properly
+    connect(m_ui->entryUrlLabel, &QLabel::linkActivated, this, [this] { m_ui->entryTabWidget->setFocus(); });
```

Comment:
Is this expected behaviour? I think I would expect the URL to remain focussed, so I can activate it again with Enter.

Reply:
It is certainly unexpected behavior for copy password button to copy the url. I tried a couple things and this is the only really elegant solution.

Reply:
Ideally the button should just place the correct text directly in the clipboard, and not rely on what happens to have focus at that moment.

Reply:
The button and the keyboard shortcut are the same thing unfortunately, so we don't have a choice in the matter.

Reply:
What happens if I do, in fact, want to copy the URL?

Reply:
You can: right click -> copy, press ctrl+u, press the copy url button in toolbar, drag select text then press ctrl+c. It only loses focus if you activate it.

Reply:
Would it make sense to handle the button and Ctrl+C shortcut separately? Then the button can always copy the password, and Ctrl+C can act contextually as it already does. I think that would be the most intuitive approach from a user perspective.

Reply:
This would still need to be fixed in this manner. Here is the problem:
1. We allow Ctrl+C to copy selected text from various text widgets to the clipboard
2. When you click the url in the preview panel it "selects" the link text
3. When you press Ctrl+C _or_ click the Copy Password button it then copies the URL because the text is selected.

You guys are vastly overthinking this issue, test this code for yourself... it literally has no adverse effect on usability.